src/coreclr/System.Private.CoreLib/src/System/RuntimeHandles.cs

```diff
@@ -1217,10 +1278,25 @@ internal static MdUtf8String GetUtf8Name(RuntimeFieldHandleInternal field)
         }
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        internal static extern FieldAttributes GetAttributes(RuntimeFieldHandleInternal field);
+        private static extern FieldAttributes GetAttributesInternal(RuntimeFieldHandleInternal field);
+
+        internal static FieldAttributes GetAttributes(RuntimeFieldHandleInternal field)
+        {
+            Debug.Assert(!field.IsNullHandle());
+            return GetAttributesInternal(field);
+        }
```

Comment:
```suggestion
        internal static extern FieldAttributes GetAttributes(RuntimeFieldHandleInternal field);
```
The assert inside the FCall should be good enough. We can omit the managed wrapper.